### PR TITLE
Group8

### PR DESCRIPTION
**Pull request by RyanHiltyAllegheny**
Here is the Group8 pull request for the Code of Conduct form. We feel we have completed our work for this section and it is ready to be merged with the main branch!

### CODE REVIEW

**Review by corlettim — CHANGES_REQUESTED** [review]
Make sure that all of your bullet points are equally spaced throughout the section. For example, some have a space between them and others do not. They should be formatted the same.

**Comment by corlettim** [comment]
Also please make sure to resolve the conflicts with the main branch :)

**Comment by RyanHiltyAllegheny** [comment]
I committed once again to make a few changes to the document, please specify what is conflicting with main if that issue continues to occur. Thanks!

**Comment by enpuyou** [comment]
> I committed once again to make a few changes to the document, please specify what is conflicting with main if that issue continues to occur. Thanks!

@RyanHiltyAllegheny I think there are merge conflicts between this branch and main in the `conduct.md` file. You would be able to see the specific conflicts once you click on the `Resolve conflicts` button

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please remove these black lines to resolve linting error

**Review by corlettim — CHANGES_REQUESTED** [review]
You still have some linting errors so make sure to fix those

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by corlettim — DISMISSED** [review]

**Comment by Mai1902** [comment]
I have just solved the linting issue, please review it again. Thank you so much!

**Comment by corlettim** [comment]
Everything looks good! Please make sure to fix the conflicts with main branch and you should be good to go

**Comment by noorbuchi** [comment]
Please resolve merge conflicts with main branch. Once that's done, I'll review and you should be good to merge once you get enough approvals.

**Review by noorbuchi — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]
The change looks good! Thanks!

**Review by corlettim — APPROVED** [review]

**Pull request merged**
